engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -142,7 +142,7 @@ <h4 translate>Content Host Properties</h4>
 
       <div class="detail">
         <span class="info-label" translate>RAM </span>
-        <span class="info-value">{{ host.facts["dmi::memory::size"] }}</span>
+        <span class="info-value">{{ (host.facts["memory::memtotal"] / 1024000).toFixed(2) + ' GB' }}</span>
```

Comment:
What about moving this to a function in the controller that is called to keep the logic out of the view layer?